config.go

```diff
@@ -81,9 +81,19 @@ func ParseConfig(data string, settings map[string]string) (Config, error) {
 		return Config{}, err
 	}
 
+	AddDefaultEnvs(&c, path)
+
 	return c, nil
 }
 
+func AddDefaultEnvs(c *Config, path string) {
```

Comment:
I think it's better to move this function somewhere in the `main.go` and call it from there. Config path is known there. 

Sth like this:
```
case CommandStop:
		if len(options.Windows) == 0 {
			fmt.Println("Terminating session...")
		} else {
			fmt.Println("Killing windows...")
		}
		config, err := GetConfig(configPath, options.Settings)       
		if err != nil {
			fmt.Fprintf(os.Stderr, err.Error())
			os.Exit(1)
		}
                SetSmugEnv(config.Session, path)
		err = smug.Stop(config, options, context)
		if err != nil {
			fmt.Fprintf(os.Stderr, err.Error())
			os.Exit(1)
		}
```